README.md

```diff
@@ -1,24 +1,58 @@
-# Detection and tracking in Python
+# Object detection and tracking in Python
 
-<p align="center">
-  <img src="https://user-images.githubusercontent.com/7695861/113501088-3590ad00-9523-11eb-81b3-448fd735375b.jpg?raw=true" alt="Example frame"/>
```

Comment:
I would prefer to keep the centre-aligned image

Reply:
HTML tags are not Markdown compliant. README.md is a markdown file and Markdown does not care for positioning or styling, hence the change.